src/urbit/sur/chess.hoon

```diff
@@ -161,6 +161,7 @@
       [%result game-id=@dau result=chess-result]
       [%draw-offer game-id=@dau]
       [%draw-declined game-id=@dau]
+      [%draw-claim-available game-id=@dau]
```

Comment:
As above, might want to change.

Reply:
See above - this was my call for merging threefold repetition and the 50-move rule into one logic flow